### DISCUSSION TITLE
是否要将自动下载Java的版本更换为21

### DISCUSSION THREAD

**Discussion by Chiloven945** [Meloong-Git/PCL#3617] [提问] [answered]
鉴于1.20.5之后需要Java21的话

**Comment by hejiehao** [comment]
<img width="741" alt="319972202-2e88eeab-ec8f-40fd-897e-f9f051321bbd" src="https://github.com/Hex-Dragon/PCL2/assets/98140773/49eb3b6f-8559-4df7-99d7-bfbb31365103">

**Comment by Pigeon0v0** [comment, marked as answer]
会根据游戏需要来决定
